api/envoy/api/v2/auth/cert.proto

```diff
@@ -216,6 +216,10 @@ message CertificateValidationContext {
 
   // If specified, Envoy will not reject expired certificates.
   bool allow_expired_certificate = 8;
+
+  // If specified, Envoy will use this regular expression while matching :ref:`Subject Alternative
```

Comment:
This setting should have the same constraint and warning as `verify_subject_alt_name`, so I think those parts of the documentation should be copied here.